lab-shelly/model/shelter.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const Cat = require('./cat');
+
+const shelterSchema = Schema({
+  name: {type: String, required: true},
+  neighborhood: {type: String, required: true},
+  cats: [{type: Schema.Types.ObjectId, ref: 'cat'}],
+});
+
+const Shelter = module.exports = mongoose.model('shelter', shelterSchema);
+
+Shelter.findByIdAndAddCat = function(id, cat) {
```

Comment:
What is this function doing? Also generally if you are going to use this with a newly instantiated shelter object, you'll want have this method be on the protoype of Shelter.